### ISSUE TITLE
Missions from 2007-2023 are missing.

### ISSUE THREAD

**Issue by alkock** [giomf/ff-eppendorf-website#33]
The old missions from 2007 til 2023 are missing.

**Comment by giomf** [comment]
Yes. Im currently working on the missions from 2016 to 2023. The problem with the  older missions is that they don't have a long text and/or do not have images, which is a bit problematic because the new web page is based on articles.

I will add them in a list, maybe. Will see.

**Comment by giomf** [comment]
Mission till 2018 have been added.
